virttest/libvirt_xml/vm_xml.py

```diff
@@ -2147,6 +2147,8 @@ def __init__(self, virsh_instance=base.virsh):
                                  tag_name='nvram')
         accessors.XMLAttribute('secure', self, parent_xpath='/',
                                tag_name='loader', attribute='secure')
+        accessors.XMLAttribute('os_firmware', self, parent_xpath='/',
+                               tag_name='os', attribute='firmware')
```

Comment:
please confirm whether tag_name is 'os' or other
moreover, please add doc description around line 2093 accordingly.

Reply:
The tag_name is 'os' and the doc description has already been added.